### PR TITLE
GDPR-10 Setting up docker build

### PR DESCRIPTION
**Pull request by brightonsbox**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -45,6 +49,35 @@ jobs:
...
+      - attach_workspace:
+          at: ~/app
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build container and publish to docker hub
+          command: |
+            APP_VERSION=$(cat ./app_version.txt)
+            docker build --rm=false . \
+               --tag "mojdigitalstudio/offender-case-notes:$APP_VERSION" \
```
Comment (petergphillips) [inline .circleci/config.yml]: offender case notes?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ENV TZ=Europe/London
+RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" > /etc/timezone
+
+RUN addgroup --gid 2000 --system appgroup && \
+    adduser --uid 2000 --system appuser --gid 2000
+
+WORKDIR /app
+
+COPY --chown=appuser:appgroup build/libs/dps-data-compliance*.jar /app/app.jar
+COPY --chown=appuser:appgroup run.sh /app
```
Comment (petergphillips) [inline Dockerfile]: missing app insights

File: .circleci/config.yml (hunk 1 above)
Comment (brightonsbox) [reply .circleci/config.yml]: Fixed

File: Dockerfile (hunk 2 above)
Comment (brightonsbox) [reply Dockerfile]: Will add as a follow up

**Pull request merged**